include/opendht/value.h

```diff
@@ -543,6 +544,9 @@ struct OPENDHT_PUBLIC Value
     Blob cypher {};
 };
 
+typedef bool (*FilterRaw)(const Value&, void *user_data);
+Value::Filter bindFilterRaw(FilterRaw raw_filter, void* user_data);
```

Comment:
Actually. I think I may need that binding function anyway since when passing `NULL` in the `get` in place of the filter, I get this:
```
building 'opendht' extension
x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fdebug-prefix-map=/build/python3.5-LIjGUm/python3.5-3.5.3=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I../../include -I../include -I/usr/include/python3.5m -c ./opendht.cpp -o build/temp.linux-x86_64-3.5/./opendht.o -std=c++11
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
In file included from ./opendht.cpp:503:0:
../include/opendht/value.h: In instantiation of ‘dht::Value::Filter::Filter(Functor) [with Functor = long int]’:
./opendht.cpp:14013:231:   required from here
../include/opendht/value.h:138:74: error: no matching function for call to ‘std::function<bool(const dht::Value&)>::function(long int&)’
         Filter(Functor f) : std::function<bool(const Value&)>::function(f) {}
                                                                          ^
In file included from /usr/include/c++/6/memory:79:0,
                 from ./opendht.cpp:493:
/usr/include/c++/6/functional:2106:7: note: candidate: template<class _Functor, class, class> std::function<_Res(_ArgTypes ...)>::function(_Functor)
       function<_Res(_ArgTypes...)>::
       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/6/functional:2106:7: note:   template argument deduction/substitution failed:
/usr/include/c++/6/functional:1913:9: error: no type named ‘type’ in ‘class std::result_of<long int&(const dht::Value&)>’
         typename = _Requires<_Callable<_Functor>, void>>
         ^~~~~~~~
/usr/include/c++/6/functional:1888:7: note: candidate: std::function<_Res(_ArgTypes ...)>::function(std::function<_Res(_ArgTypes ...)>&&) [with _Res = bool; _ArgTypes = {const dht::Value&}]
       function(function&& __x) : _Function_base()
       ^~~~~~~~
/usr/include/c++/6/functional:1888:7: note:   no known conversion for argument 1 from ‘long int’ to ‘std::function<bool(const dht::Value&)>&&’
/usr/include/c++/6/functional:2092:5: note: candidate: std::function<_Res(_ArgTypes ...)>::function(const std::function<_Res(_ArgTypes ...)>&) [with _Res = bool; _ArgTypes = {const dht::Value&}]
     function<_Res(_ArgTypes...)>::
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/6/functional:2092:5: note:   no known conversion for argument 1 from ‘long int’ to ‘const std::function<bool(const dht::Value&)>&’
/usr/include/c++/6/functional:1868:7: note: candidate: std::function<_Res(_ArgTypes ...)>::function(std::nullptr_t) [with _Res = bool; _ArgTypes = {const dht::Value&}; std::nullptr_t = std::nullptr_t]
       function(nullptr_t) noexcept
       ^~~~~~~~
/usr/include/c++/6/functional:1868:7: note:   no known conversion for argument 1 from ‘long int’ to ‘std::nullptr_t’
/usr/include/c++/6/functional:1861:7: note: candidate: std::function<_Res(_ArgTypes ...)>::function() [with _Res = bool; _ArgTypes = {const dht::Value&}]
       function() noexcept
       ^~~~~~~~
/usr/include/c++/6/functional:1861:7: note:   candidate expects 0 arguments, 1 provided
error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
Makefile:556: recipe for target 'pybuild.stamp' failed
make[2]: *** [pybuild.stamp] Error 1
make[2]: Leaving directory '/home/simon/prog/opendht/python'
Makefile:503: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/simon/prog/opendht'
Makefile:412: recipe for target 'all' failed
make: *** [all] Error 2
```